doc/getting-started.md

```diff
@@ -21,59 +19,72 @@ deployment "istio-mixer" created
 ## Connecting microservices with Istio
 
 This guide shows how to set up Istio and manipulate the proxy mesh to achieve useful behavior.
-In this example we use two microservices:
+In this example we use *CouchDB* and *NGINX* as and demonstrate how to manipulate them:
```

Comment:
Can we use something other than Nginx? Nginx will typically confuse the end user, as its perceived as a reverse proxy. Why chain two reverse proxies? A simpler option would be a python or nodejs service.